src/daft-core/src/series/ops/is_in.rs

```diff
@@ -0,0 +1,53 @@
+use common_error::DaftResult;
+
+use crate::{
+    array::ops::DaftIsIn, datatypes::BooleanArray, with_match_comparable_daft_types, DataType,
+    IntoSeries, Series,
+};
+
+#[cfg(feature = "python")]
+use crate::series::ops::py_membership_op_utilfn;
+
+impl Series {
+    pub fn is_in(&self, items: &Self) -> DaftResult<Series> {
+        let default =
```

Comment:
you should only create `default` if you actually want to return it. Otherwise you always have this allocation